golang/cosmos/x/vstorage/types/types.go

```diff
@@ -7,3 +7,21 @@ func NewData() *Data {
 func NewChildren() *Children {
 	return &Children{}
 }
+
+type StorageEntry []string
+
+func (sc StorageEntry) IsPresent() bool {
+	return len(sc) == 2
```

Comment:
I prefer a little bit of defensiveness here:
```suggestion
	return len(sc) >= 2
```